src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -242,6 +242,16 @@ class ngraph::runtime::interpreter::INTExecutable : public Executable
     {
         const Node& node = *node_wrapper.get_node();
 
+        size_t opset_version = node.get_opset_version();
+        if (opset_version != 0)
```

Comment:
What happens if:

- INTERPRETER supports opset v5, so this check is changed to `!= 5`, but
- The op `Reshape` (for example) has not had its behavior changed since v4?

If the answer is "that's okay, because the class will have its version number bumped up to 5 even though the behavior has changed", what happens to things that only support v4? (Also, in that case, I think the checks in `Opset1Transformation` would become invalid.)

Reply:
BTW, let's use `NGRAPH_CHECK`.

Reply:
`NGRAPH_CHECK` was added.
I think that the question about offset support is key.
I understood that we update version of op: for instance from `v0` to `v1` only when specification has been changed. In such case I agree with you that the condition is incorrect.  
I aware that it is not very flexible but maybe each op in each version should have list of supported backends (or unsupported)?